templates/webapi/main/index.html.ep

```diff
@@ -83,20 +83,11 @@
 
             <div class="form-group">
                 <strong>
-                    Auto refresh
-                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically</p>') %>
+                    Auto refresh in (seconds)
+                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically. To stop the automatic refresh select 0</p>') %>
                 </strong>
+                <input type="number" class="form-control" name="interval" value="0" id="filter-limit-builds">
```

Comment:
It would be nicer to use a placeholder text instead of "0" and to append the extension like in the size limit for assets input (e.g. https://openqa.opensuse.org/admin/job_templates/1).

```suggestion
                    Auto-refresh interval
                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically. To stop the automatic refresh select 0</p>') %>
                </strong>
                <input type="number" class="form-control" name="interval" id="filter-limit-builds" placeholder="no auto-refresh"> seconds
```